### ISSUE TITLE
Inference speed

### ISSUE THREAD

**Issue by sarahmhooper** [yransun/DIFR3CT#1]
Hi, thank you for the interesting paper, I'm looking forward to the code! I'm wondering what the inference speed is? For example, to go from one set of x-rays to a fully reconstructed CT. I saw a detail on training time in the paper, but I'm curious for test time. Thanks again!

**Comment by yransun** [comment]
Hi, thanks for your interest! 

We tested two optional samplers in our code during inference. Considering the inputs are two biplanar X-rays, if we chose regular "step by step" denoising (T = 1000 steps), the average time for inferring one 3D reconstructed CT is about 167 seconds; if we chose DPM-Solver++ (https://github.com/LuChengTHU/dpm-solver), and set T = 10 steps, the average time for inferring one 3D reconstructed CT is about 2.13 seconds. When using DPM-Solver++, the T can be set larger like 20 or 50 steps. We noticed that the number of input X-rays influence the inference time quite small. And I think maybe other faster samplers can also be easily used in our code without retraining the model.

We did the experiments on NVIDIA A100, the speed might be influenced by how busy the cores are.

Feel free to let me know if you have any other questions :)
